### PR TITLE
Address typography in interconnects

### PR DESCRIPTION
**Pull request by eduardoj**
**Before**

Screenshot from 2019-08-20 17-30-05

**After**

Screenshot from 2019-08-20 17-30-29

### CODE REVIEW

**Comment by eduardoj** [comment]
[Address typography - Configuration](https://trello.com/c/A1DIEngF/1214-address-typography-configuration)

File: src/api/app/views/webui2/webui/interconnects/_interconnect_button.html.haml [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
                            disable_with: 'Connecting...',
                            method: :post },
-                   class: 'btn btn-sm btn-primary interconnect' do
+                   class: 'btn btn-primary interconnect' do
         Connect
-  .interconnect-info.text-muted
-    %i.fa.fa-cubes.fa-xs{ class: "#{'text-secondary' if exists}" }
+  %div{ class: "#{'text-muted' unless exists}" }
+    %i.fa.fa-cubes{ class: "#{'text-secondary' if exists}" }
     - if exists
```
Comment (dmarcoux) [inline src/api/app/views/webui2/webui/interconnects/_interconnect_button.html.haml]: I suggest replacing the `if/else` by `link_to_if`:

`= link_to_if(exists, name, project_show_path(project: name))`

Comment (eduardoj) [reply src/api/app/views/webui2/webui/interconnects/_interconnect_button.html.haml]: Nice! Added.

**Comment by dmarcoux** [comment]
There's a typo in the commit message: We can make use of <strike>a</strike> this Rails helper to refactor the code.

Otherwise, it's all good!

**Comment by eduardoj** [comment]
> There's a typo in the commit message: We can make use of a this Rails helper to refactor the code.

Fixed. Thanks!

**Pull request merged**